decidim-admin/app/views/decidim/admin/newsletters/index.html.erb

```diff
@@ -49,6 +49,10 @@
                   <%= icon_link_to "pencil", decidim_admin.edit_newsletter_path(newsletter), t("actions.edit", scope: "decidim.admin"), class: "edit" %>
                 <% end %>
 
+                <% if allowed_to?(:read, :newsletter, newsletter: newsletter) %>
+                  <%= icon_link_to "share", decidim_admin.send_to_user_newsletter_path(newsletter), t("actions.send_me_a_test_email", scope: "decidim.admin"), class: "preview", method: :post %>
```

Comment:
Now you need to add a test that the admin clicks on this button and receives the test newsletter

Reply:
Thanks Ivan. It's done.